src/js/controller/model.ts

```diff
@@ -177,6 +177,9 @@ class Model extends SimpleModel {
             this._provider.destroy();
             this._provider = null;
         }
+        if (this.mediaModel) {
+            this.mediaModel.off();
+        }
```

Comment:
@david-almaguer I added this change in response to JW8-11904 (removes "change:duration" and "change:mediaState" listeners).